proxy/operator_resource/resource_manager.py

```diff
@@ -121,6 +129,53 @@ def free_resource(self, is_good_resource: bool, op_resource: OpResourceModel) ->
                 _LOG.debug("disable resource: %s", op_resource)
                 op_signer.disabled_holder_list.append(op_holder)
 
+            try:
+                self._stat_client.commit_op_resource_holder_status(
+                    OpResourceHolderStatusData(
+                        owner=op_resource.owner,
+                        free_holder_cnt=len(op_signer.free_holder_list),
+                        used_holder_cnt=len(op_signer.used_holder_dict),
+                        disabled_holder_cnt=len(op_signer.disabled_holder_list),
+                    )
+                )
+
+                if not await self._validate_op_balance(op_signer):
+                    if op_signer := self._active_signer_dict.pop(op_signer.owner, None):
+                        self._disabled_signer_dict[op_signer.owner] = op_signer
+
+                spending_tokens_balance, self._spending_tokens_balance = self._spending_tokens_balance, 0
+
+                self._stat_client.commit_op_resource_spending_tokens_balance(
+                    OpResourceSpendingTokensBalanceData(owner=op_resource.owner, balance=spending_tokens_balance)
+                )
+
+                evm_cfg = await self._server.get_evm_cfg()
+                token_name = evm_cfg.chain_dict[op_resource.chain_id].name
+
+                neon_address = NeonAccount.from_raw(op_resource.eth_address, op_resource.chain_id)
+                neon_account = await self._core_api_client.get_neon_account(neon_address)
+                self._stat_client.commit_op_resource_earned_tokens_balance(
+                    OpResourceEarnedTokensBalanceData(
+                        token_name=token_name,
+                        eth_address=op_resource.eth_address,
+                        balance=neon_account.balance,
```

Comment:
Ah. My intention was just in case the `NeonAccountModel.eth_address` is different to the `OperatorAccountModel.eth_address`.

Combined now, which will be less confusing for operators.